### ISSUE TITLE
Compile errors with Code::Blocks 15.12

### ISSUE THREAD

**Issue by crazyafrican** [jupe/gppanel#8]
I get these errors when compiling with the supplied mingw on Code::Blocks 15.12
```
..\include\Fourier.inline.h|332|error: lvalue required as left operand of assignment|
..\include\Fourier.inline.h|333|error: lvalue required as left operand of assignment|
```
I'm guessing it's the version of mingw that I'm using. wxWidgets 3.0.2 also forces me to compile with
```
-std=c++11
```
I also had to manually define M_PI_2 for some reason

Thanks

**Comment by shaiku** [comment]
This can be fixed without going to c++11 by changing these lines:
```
data[lIncrement].real() = cosA * temp.real() + sinA * temp.imag();
data[lIncrement].imag() = cosA * temp.imag() - sinA * temp.real();
```

to:
`data[lIncrement] = std::complex<T>(cosA * temp.real() + sinA * temp.imag(), cosA * temp.imag() - sinA * temp.real());`

In fourier.inline

**Comment by jupe** [comment]
merged PR #9 should include fix for this.
